### PR TITLE
support for multihosts

### PR DESCRIPTION
**Pull request by SergeyPirogov**

### CODE REVIEW

**Comment by SergeyPirogov** [comment]
We faced this problem in our infrastructure when selenoid hosts have different ips and there is no possibility to set dns.

I fixed template and added new defult vars:
This playbook will generate valid selenoid.xml see example:

```
---
- hosts: all
  vars:
    grid_router_path: "{{ ansible_env.HOME }}/grid-router"
    grid_router_qouta_path: "{{ ansible_env.HOME }}/grid-router/quota"
    grid_router_port: 4445
    
    grid_router_regions:
        - name: "region-1"
          hosts:
          - name: 172.17.0.2
            port: 4444
            browser_count: 4
          - name: 145.239.131.73
            port: 4445
            browser_count: 4            
            
    grid_router_browsers:
        - name: "chrome"
          defaultVersion: "62.0"
          versions:
            - "62.0"
            - "63.0"
  roles:
    - gridrouter-docker
```


```
<qa:browsers xmlns:qa="urn:config.gridrouter.qatools.ru">
    <browser name="chrome" defaultVersion="62.0">
        <version number="62.0">
            <region name="region-1">
                <host name="172.17.0.2" port="4444" count="4"></host>
                <host name="145.239.131.73" port="4445" count="4"></host>
            </region>
        </version>
        <version number="63.0">
            <region name="region-1">
                <host name="172.17.0.2" port="4444" count="4"></host>
                <host name="145.239.131.73" port="4445" count="4"></host>
            </region>
        </version>
    </browser>
</qa:browsers>
```

File: README.md [hunk 1]
```diff
@@ -72,8 +79,29 @@ Ggr is [using](http://aerokube.com/ggr/latest/#_creating_users_file) htpasswd fi
...
+    grid_router_qouta_path: "{{ ansible_env.HOME }}/grid-router/quota"
+    grid_router_port: 4445
+    
+    grid_router_regions:
+        - name: "region-1"
+          hosts:
+          - name: 172.17.0.2
+            port: 4444
+            browser_count: 4
+          - name: 145.239.131.73
```
Comment (arslanbekov) [inline README.md]: it's better to put a stub :)

**Comment by arslanbekov** [comment]
Thank you!

**Comment by arslanbekov** [comment]
Fixed #11

**Pull request merged**